### ISSUE TITLE
Crash when creating a new world or opening a world.

### ISSUE THREAD

**Issue by EnderT5** [Shinoow/AbyssalCraft#88]
http://pastebin.com/gujLspkw

it happens at evilcraft and biomes o plenty too (biome adding mods) but not integrated dynamics (another biome adding mod, works fine)

**Comment by Shinoow** [comment]
A mod (no idea which) is initializing their EnumCreatureType incorrectly. I would start taking out mods until you find which one's causing it (while keeping AbyssalCraft, EvilCraft or BoP, as they are triggered), then report to the author of that mod. Show them this: https://github.com/TeamLapen/Vampirism/issues/64#issuecomment-215508278

**Comment by EnderT5** [comment]
i havent got wampirism, what can it be? and i dont understand anything about enumcreaturetype, please tell me what is it and i will test this mods one by one, instead of testing all mods one by one.

**Comment by Shinoow** [comment]
Well, firstly, the thing I pointed to you is what needs to be done by the developer of the mod responsible for this. Vampirism had the same issue, but that was solved (hence there being a solution).
EnumCreatureTypes are used to determine where something spawns (you assign one when you decide in which biomes your mob should spawn, so that the game can pick your mob depending on conditions). For example there's the `CREATURE` and `MONSTER` types, where `CREATURE` is treated as an animal, making them spawn the way that normal animals spawn, while `MONSTER` makes the game spawn your mob the way mobs are spawned (in caves, dark places or outside during night).
